datamarket/components/MyDatasets/ListDatasets.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { Dataset, DatasetVisibility } from "../../constants";
+import AddIcon from '@mui/icons-material/Add';
+import { DatasetView } from "./DatasetView";
+import { useRouter } from "next/router";
+import SettingsIcon from '@mui/icons-material/Settings';
+import styles from './ListDatasets.module.css';
+import ClearIcon from '@mui/icons-material/Clear';
+import {SideBar} from "./SideBar";
+
+
+export const ListDatasets = ({
```

Comment:
adding 
```
/**
* some info about what the different params are
**/
```
is useful to define params when theyre not super obvious
ie why is liteMode useful.

Reply:
Done